NBitcoin/Script.cs

```diff
@@ -891,7 +891,7 @@ public static bool VerifyScript(Script scriptSig, Script scriptPubKey, Transacti
 
 #if !NOCONSENSUSLIB
 
-		public const string LibConsensusDll = "libbitcoinconsensus.dll";
+		public const string LibConsensusDll = "./libbitcoinconsensus.dll";

```

Comment:
I would suggest trying just [DllImport("bitcoinconsensus")]. I believe MS.NET has default logic of trying "lib" and ".dll" prefixes, while Mono uses .so for Linux and .dylib on macOS.

Reply:
I meant `lib` prefix and `.dll` suffix.